src/main/resources/swagger/api-docs.yaml

```diff
@@ -530,6 +530,427 @@ paths:
           - email
           - profile
 
+  /api/workflows/{version}/{id}/abort:
```

Comment:
If we plan to remove these in the future, maybe we shouldn't publish them in the swagger interface, or we should mark them as deprecated to discourage anyone else from discovering and using them.

Reply:
I think having them in Swagger is valuable for our own documentation and trying out requests, but I agree we could use a "Job Manager only for now" notice. I'll see what I can do.

Reply:
OOI, what makes you think we'd remove them in the future?

Reply:
I put a comment in the code saying if we ever go through the work to route Job Manager directly to CromIAM, we should clean up these routes